.github/workflows/actions/publish-npm-package/action.yml

```diff
@@ -58,20 +92,25 @@ runs:
       echo "List '@${{ inputs.scope }}/${{ inputs.package }}' package"
       ls -al ${{ inputs.package }}/pkg
 
-  - name: Test publish package
-    if: inputs.dry_run == 'true' && steps.check_version.outputs.should_deploy == 'true'
+  - name: Publish package new version
+    if: steps.check_version.outputs.deploy_mode == 'publish'
     shell: bash
     env:
       NPM_TOKEN: ${{ inputs.api_token }}
     run: |
-      echo "test publish '@${{ inputs.scope }}/${{ inputs.package }}' package"
+      echo "Publish '@${{ inputs.scope }}/${{ inputs.package }}' package"
       npm set "//registry.npmjs.org/:_authToken=${NPM_TOKEN}"
```

Comment:
Can we check if the `NPM_TOKEN` is available and display a warning message when not available instead of running the `npm whoami` that will systematically fail (this occurs when a PR is coming from a fork)?